localization/kitti_evaluator/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kitti_evaluator</name>
```

Comment:
If this tool is just comparing localization output and ground truth values, then I think we can use more generic name such as localization_evaluator